proto/ocpaths.proto

```diff
@@ -62,6 +75,24 @@ message OCPath {
   // GNMIRpc describes expected (or supported) behavior for a particular
   // Openconfig path.
   GNMIRpc gnmi_rpc = 5;
+
+  // the date/time when this path became supported. not present if 
+  // unknown.
+  optional Release origination = 6;
+
+  // the date/time when this path became deprecated. not present for active
```

Comment:
how "active" is defined in this context?

aren't deprecated path still usable?

Reply:
ah, I needed a word for not-deprecated and made one up (without definition). 
I'll clear this comment up.